commands_to_load/Mod.py

```diff
@@ -82,5 +83,460 @@ def __init__(self, bot):
+        await ctx.message.delete()
+        logger.info('[Mod mute()] invoking command deleted')
+        
+        if not ctx.message.author in discord.utils.get(ctx.guild.roles, name="Minions").members:
+            logger.info('[Mod mute()] unathorized command attempt detected. Being handled.')
+            await self.rekt(ctx)
+            return
+
+        # Check for mention
+        # If there get user
+        logger.info('[Mod mute()] checking for mention')
+        mentions = ctx.message.mentions
+        if not mentions:
+            logger.info('[Mod mute()] No mention found. Informing user')
+            eObj = await em(ctx, description='You need to @ mention the user to mute', footer='Invalid arguments')
+            if eObj is not False:
+                await ctx.send(embed=eObj, delete_after=5.0)
+            return
+
+        if len(mentions) > 1:
+            logger.info('[Mod Mute()] more than 1 mention. Informing user')
+            eObj = await em(ctx, description='You have too many @\'s. Please only @ mention one target user.', footer='Invalid arguments')
+            if eObj is not False:
+                await ctx.send(embed=eObj, delete_after=5.0)
+            return
+        
+        user = mentions[0]
+        logger.info('[Mod mute()] user found through mention: {}'.format(user))
+
+        # Grab the Muted role
+        MUTED_ROLE = discord.utils.get(ctx.guild.roles, name='Muted')
+        
+        # Check if muted role is there
+        if not MUTED_ROLE:
+            eObj = await em(ctx, description='Muted role is missing', footer='Command error')
+            if eObj is not False:
+                await ctx.send(embed=eObj, delete_after=5.0)
+            return
+        
+        logger.info('[Mod mute()] mute role found: {}'.format(MUTED_ROLE.id))
+
+        # Add muted role to user
+        await user.add_roles(MUTED_ROLE)
+        logger.info('[Mod mute()] adding muted role to user')
+
+        # Tell them in dm
+        logger.info('[Mod mute()] informing {} that they are muted in dm'.format(user))
+        eObj = await em(ctx, title='Moderator action was taken against you', colour=0xff0000, description='You\'ve been muted in the CSSS server. Message a minion to learn why and how to be unmuted', footer='Moderator action')
+        if eObj is not False:
+            await user.send(embed=eObj)
+
+        # Tell council of action 
+        logger.info('[Mod mute()] getting council channel')
+        council = discord.utils.get(ctx.guild.channels, name='council')
+        logger.info('[Mod mute()] council channel found: {}'.format(council.id))
+
+        logger.info('[Mod mute()] informing council of {}\'s action to mute {}'.format(ctx.message.author, user))
+        eObj = await em(ctx, description='{} muted {}'.format(ctx.message.author, user))
+        if eObj is not False:
+            await council.send(embed=eObj)
+
+    @commands.command()
+    async def unmute(self, ctx):

```

Comment:
encountered this issue

2019-04-16 03:28:58 = INFO - [Mod unmute()] telling council of the unmuting
2019-04-16 03:28:58 = WARNING - Traceback (most recent call last):
2019-04-16 03:28:58 = WARNING -
2019-04-16 03:28:58 = WARNING -   File "/usr/local/lib/python3.5/site-packages/discord/ext/commands/core.py", line 62, in wrapped
.    ret = await coro(*args, **kwargs)
2019-04-16 03:28:58 = WARNING -
2019-04-16 03:28:58 = WARNING -   File "/usr/src/app/commands_to_load/Mod.py", line 448, in unmute
.    await council.send(embed=eObj)
2019-04-16 03:28:58 = WARNING -
2019-04-16 03:28:58 = WARNING -   File "/usr/local/lib/python3.5/site-packages/discord/abc.py", line 745, in send
.    embed = embed.to_dict()
2019-04-16 03:28:58 = WARNING -
2019-04-16 03:28:58 = WARNING - AttributeError: 'coroutine' object has no attribute 'to_dict'
2019-04-16 03:28:58 = WARNING -
2019-04-16 03:28:58 = WARNING -
The above exception was the direct cause of the following exception:
2019-04-16 03:28:58 = WARNING -
2019-04-16 03:28:58 = WARNING - Traceback (most recent call last):
2019-04-16 03:28:58 = WARNING -
2019-04-16 03:28:58 = WARNING -   File "/usr/local/lib/python3.5/site-packages/discord/ext/commands/bot.py", line 898, in invoke
.    await ctx.command.invoke(ctx)
2019-04-16 03:28:58 = WARNING -
2019-04-16 03:28:58 = WARNING -   File "/usr/local/lib/python3.5/site-packages/discord/ext/commands/core.py", line 611, in invoke
.    await injected(*ctx.args, **ctx.kwargs)
2019-04-16 03:28:58 = WARNING -
2019-04-16 03:28:58 = WARNING -   File "/usr/local/lib/python3.5/site-packages/discord/ext/commands/core.py", line 71, in wrapped
.    raise CommandInvokeError(exc) from exc
2019-04-16 03:28:58 = WARNING -
2019-04-16 03:28:58 = WARNING - discord.ext.commands.errors.CommandInvokeError: Command raised an exception: AttributeError: 'coroutine' object has no attribute 'to_dict'
2019-04-16 03:28:58 = WARNING -
2019-04-16 03:28:58 = WARNING - /usr/local/lib/python3.5/asyncio/tasks.py:240: RuntimeWarning: coroutine 'embed' was never awaited
.  result = coro.send(None)